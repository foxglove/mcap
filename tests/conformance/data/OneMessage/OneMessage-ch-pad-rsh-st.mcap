version https://git-lfs.github.com/spec/v1
oid sha256:9742bbf3e434f1577e5d382d3be3d7571cbec1694fc5de63818b6a1c3e714da0
size 350
