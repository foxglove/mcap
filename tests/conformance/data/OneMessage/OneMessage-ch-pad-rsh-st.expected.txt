Header library="" profile=""
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
MessageIndex channel_id=1 records={2=79}
Schema id=1 schema=b schema_encoding=c schema_name=Example
Statistics attachment_count=0 channel_count=1 channel_message_counts={1=1} chunk_count=1 message_count=1
ChunkIndex chunk_length=160 chunk_start_offset=28 compressed_size=113 compression="" end_time=2 message_index_length=34 message_index_offsets={1=190} start_time=2 uncompressed_size=113
Footer summary_crc=0 summary_offset_start=0 summary_start=240
